manifests/service.pp

```diff
@@ -10,9 +10,8 @@
   systemd::unit_file { 'puppet_webhook.service':
```

Comment:
Unrelated, but you can let this define the service if you pass in `enable` or `active` which also has the correct notify.